docs/api-queries.md

```diff
@@ -26,9 +26,26 @@ present.
 
 ### queryAllBy
 
-`queryAllBy*`queries return an array of all matching nodes for a query, and
+`queryAllBy*` queries return an array of all matching nodes for a query, and
 return an empty array (`[]`) if no elements match.
 
+### findBy
+
+`findBy*` queries return a promise which resolves when an element is found which
+matches the given query. The promise is rejected if no element is found after
+a default timeout of `4500`ms.
+
+> Note, this is a simple combination of `getBy*` queries and
+> [`waitForElement`](/docs/api-async#waitforelement). The `findBy*` queries
+> accept the `waitForElement` options as the last argument. (i.e.
+> `findByText(container, 'text', queryOptions, waitForElementOptions)`)
+
+### findAllBy
+
+`findAllBy*` queries return a promise which resolves to an array of elements when
+any elements are found which match the given query. The promise is rejected if no
+elements are found after a default timeout of `4500`ms.
+
```

Comment:
Can you add `findBy` and `findAllBy` to the notes below the headings, for Ctrl-F-ability?

> getByLabelText, queryByLabelText, getAllByLabelText, queryAllByLabelText, findByLabelText, findAllByLabelText

Reply:
Thanks! Done.